README.md

```diff
@@ -1,24 +1,153 @@
-# README
+# Greetings ![Rails](https://rubyonrails.org/assets/images/logo.svg)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Database](#database)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
**OPTIONAL**- Though your project works fine, but there are some improvements to e made to help make your project easier to read and maintain.
**OPTIONAL** 1. You have your redux actions, reducer function, and thunkcreators in the `store,js` file, it is advisable to put your redux functionality in a different file from the `store.js` file, this would help make your project more modular. Please fix this 😄 
**OPTIONAL** 2. I suggest that you use the redux toolkit in your redux store, this would help you avoid a lot of boilerplate code that comes with using redux, please fix this 🙏 
![5yw5ywy](https://github.com/Metaverse-Nabeel/greetings/assets/73607512/823108fa-970b-4344-a64d-79f720d750f2)